spiffworkflow-frontend/test/browser/helpers.py

```
@@ -0,0 +1,61 @@
+from playwright.sync_api import Page
+
+def print_page_details(page: Page):
```

Pylint (3.3.7)

[refactor] Too many branches (16/12)

(R0912)